docs/api.yml

```diff
@@ -500,6 +511,18 @@ components:
     JSONNote:
       type: object
       properties:
+        id: 
+          type: string
+          example: "149f72cb93e345708cf924e432d4f996"
+        created_date:
+          type: string
+          format: date
+        analysis_date:
+          type: string
+          format: date-time
+        event:
```

Comment:
As far as I can tell, by examining data in Elasticsearch, `event` is never used (certainly this field doesn't exist in any routed notifications.  
So we should probably say, at least for Outgoing notifications that this is likely to be absent.